index.bs

```diff
@@ -2830,19 +2835,19 @@ the same procedure as other [=assertion signatures=] generated by the [=authenti
                 <td> [=multi-factor capable|Multi-factor=] </td>
             </tr>
             <tr>
-                <th> <dfn>First-factor platform authenticator</dfn> </th>
+                <th> First-factor platform authenticator </th>
                 <td> [=platform attachment|platform=] </td>
                 <td> [=client-side credential storage modality|Client-side storage=] </td>
                 <td> [=multi-factor capable|Multi-factor=] </td>
             </tr>
             <tr>
-                <th> <dfn>Second-factor roaming authenticator</dfn> </th>
+                <th> Second-factor roaming authenticator </th>
                 <td> [=cross-platform attachment|cross-platform=] </td>
                 <td> [=server-side credential storage modality|Server-side storage=] </td>
                 <td> [=single-factor capable|Single-factor=] </td>
             </tr>
             <tr>
-                <th> <dfn>User-verifying roaming authenticator</dfn> </th>
+                <th> User-verifying roaming authenticator </th>
```

Comment:
I'm tempted to leave these dfns here and export them. In any case, we ought to be exporting a fair number of our dfns I'm thinking. see also issue #1049.

Reply:
I'm not sure where these definitions would ever be used outside this spec. How about removing them for now and instead documenting in #1049 that we might want to consider re-introducing them later?

Reply:
sure :)